README.md

```diff
@@ -55,7 +55,7 @@ For Scala/Java applications using SBT/Maven project definitions, link your appli
 
     groupId = com.microsoft.azure
     artifactId = azure-eventhubs-spark_2.11
-    version = 2.3.13
+    version = 2.3.14.1
```

Comment:
Let's keep 2.3.13 as a recommended version here. We will need to release a new version which fixes receiver close issue that was introduced 2.3.14 and update the version accordingly.